frontend/src/lib/components/LemonCalendar/LemonCalendar.tsx

```diff
@@ -0,0 +1,141 @@
+import './LemonCalendar.scss'
+import React, { useEffect, useState } from 'react'
+import { dayjs } from 'lib/dayjs'
+import { range } from 'lib/utils'
+import { LemonButton, LemonButtonProps } from 'lib/components/LemonButton'
+import { IconChevronLeft, IconChevronRight } from 'lib/components/icons'
+import clsx from 'clsx'
+
+export interface LemonCalendarProps {
+    /** Fired if a calendar cell is clicked */
+    onClick?: (date: string) => void
```

Comment:
if this was `onCellClick` you wouldn't need the comment and the caller would have a clearer signal of when this fires...